src/gc/cdgc/gc.d

```diff
@@ -43,7 +43,6 @@ version = STACKGROWSDOWN;       // growing the stack means subtracting from the
 /***************************************************/
 
 import gc.cdgc.bits: GCBits;
-import gc.cdgc.stats: GCStats, Stats;
```

Comment:
This module is not removed. Honestly I like the stats hooks that we had in place here. Also that stats are collected in a separated struct instead of as a regular GC member variable.

Also in the commit title (Replace old stats system with plan gc_stats) there is a typo "plan".